control_mode:
    J1_A: 0
    J_wheel_A: 0
    J1_B: 0
    J_wheel_B: 0
    J1_C: 0
    J_wheel_C: 0
    J1_D: 0
    J_wheel_D: 0
    J1_E: 3 # 7 pos + vel + effort # 3 pos + vel #1 pos
    J2_E: 3 # 7 pos + vel + effort # 3 pos + vel #1 pos
    J3_E: 3 # 7 pos + vel + effort # 3 pos + vel #1 pos
    J4_E: 3 # 7 pos + vel + effort # 3 pos + vel #1 pos
    J5_E: 3 # 7 pos + vel + effort # 3 pos + vel #1 pos
    J6_E: 3 # 7 pos + vel + effort # 3 pos + vel #1 pos
    J7_E: 3 # 7 pos + vel + effort # 3 pos + vel #1 pos

#    'J1_B': *high_stiff
#    'J1_C': *high_stiff
#    'J1_D': *high_stiff
#    'J1_E': *high_stiff
#    'J2_E': *high_stiff
#    'J3_E': *high_stiff
#    'J4_E': *high_stiff
#    'J5_E': *high_stiff
#    'J6_E': *high_stiff

#fixed_joints:
#    J1_A: 0.0
#    J_wheel_A: 0.0
#    J1_B: 0.0
#    J_wheel_B: 0.0
#    J1_C: 0.0
#    J_wheel_C: 0.0
#    J1_D: 0.0
#    J_wheel_D: 0.0

define:
#    - &low_stiff 0.
#    - &low_damp 0.
    - &low_stiff 50.
    - &low_damp 5.
    - &high_stiff 1000.
    - &high_damp 10.

#stiffness:
#    'J1_A': *high_stiff
#    'J1_B': *high_stiff
#    'J1_C': *high_stiff
#    'J1_D': *high_stiff
#    'J1_E': *high_stiff
#    'J2_E': *high_stiff
#    'J3_E': *high_stiff
#    'J4_E': *high_stiff
#    'J5_E': *high_stiff
#    'J6_E': *high_stiff
#    J_wheel_A: 0
#    J_wheel_B: 0
#    J_wheel_C: 0
#    J_wheel_D: 0
#
#damping:
#    'J1_A': *high_damp
#    'J1_B': *high_damp
#    'J1_C': *high_damp
#    'J1_D': *high_damp
#    'J1_E': *high_damp
#    'J2_E': *high_damp
#    'J3_E': *high_damp
#    'J4_E': *high_damp
#    'J5_E': *high_damp
#    'J6_E': *high_damp
#    J_wheel_A: 0
#    J_wheel_B: 0
#    J_wheel_C: 0
#    J_wheel_D: 0

